examples/cpp/CPUDistribution.cc

```diff
@@ -14,17 +14,27 @@
 #include <cstdlib>
 #include <iomanip>
 #include <iostream>
+#include <memory>
 #include <string>
 
 #include "BPF.h"
+#include "shared_table.h"
+#include "table_desc.h"
+#include "common.h"
+
+const std::string BPF_TABLES = R"(
```

Comment:
i don't understand the purpose of this split. are you suggesting that all .py scripts would need to split table def out of c code?